From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Fri, 11 Mar 2022 12:46:16 -0500
Subject: [PATCH] Rename "dprintf' to "dbgprintf"

stdio defines a dprintf() macro now, so using dprintf() for our debug
printer gets obnoxious warnings.  This renames it to dbgprintf().

Signed-off-by: Peter Jones <pjones@redhat.com>
---
 src/cms_common.c    | 73 +++++++++++++++++++++++++++++------------------------
 src/cms_pe_common.c | 20 +++++++--------
 src/efikeygen.c     | 16 ++++++------
 src/file_pe.c       |  6 +++--
 src/password.c      | 68 ++++++++++++++++++++++++-------------------------
 src/pesign.c        | 10 ++++----
 src/util.h          | 26 +++++++++----------
 7 files changed, 114 insertions(+), 105 deletions(-)

NOTE: This change is already present in pesign 116. The patch body is a no-op so the packaging series continues to apply cleanly after the version bump.

diff --git a/TODO b/TODO
--- a/TODO
+++ b/TODO
@@ -1 +1 @@
-Stuff that needs to get done:
+Stuff that needs to get done:
